test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+	
+	private Name testName;
+	private Name testName2;
+	
+	@Before
+	public void setup() throws IllegalValueException {
+		testName = new Name("Obama");
+		testName2 = new Name("obama");
```

Comment:
Good usage of `@Before`. You may want to rename your test class to a more specific one as other testing scenarios may not benefit from these class variables.